src/site/_includes/js/eligibility.js

```diff
@@ -155,12 +160,6 @@ const cnst = {
   housingChoice: {
     // https://www.scchousingauthority.org/wp-content/uploads/2022/08/Eng-_Interest_List_Flyer.pdf
     MIN_ELIGIBLE_AGE: 18,
-    // For income limits additional person calculations
-    // https://www.huduser.gov/portal/datasets/il/il2023/2023IlCalc.odn?inputname=Santa+Clara+County&area_id=METRO41940M41940&fips=0608599999&type=county&year=2023&yy=23&stname=California&stusps=CA&statefp=06&ACS_Survey=%24ACS_Survey%24&State_Count=%24State_Count%24&areaname=San+Jose-Sunnyvale-Santa+Clara%2C+CA+HUD+Metro+FMR+Area&incpath=%24incpath%24&level=50
-    INCOME_ROUND_UP_TO_NEAREST: 50, // USD
-    BASE_HOUSEHOLD_SIZE: 4, // People
-    FAMILY_SIZE_ADJ_8: 1.32,
-    INCREMENTAL_ADJ: 0.08,
     // https://www.ecfr.gov/current/title-24/subtitle-B/chapter-IX/part-982#p-982.201(b)(1)(i)
```

Comment:
Should these comments be removed? Or moved to `cnst.common`?

Reply:
The ecfr.gov, "very low income" and associated huduser.gov comments are all to do with the actual Housing Choice Voucher income limits themselves (64550, 73750, etc) and so should stay in the `housingChoice` object.  

The comments to do with the large family size calculations have already been moved into `common` (e.g.

Reply:
I see, thanks for clarifying!